documentation/src/main/asciidoc/topics/con_file_based_cache_stores.adoc

```diff
@@ -1,72 +1,33 @@
-
-Additionally if you attempt to use transactional caches with shared file systems, unrecoverable failures can happen when writing to files during the commit phase.
-====
-
-[discrete]
-== Soft-Index File Stores
-
-`SoftIndexFileStore` is the default implementation for file-based cache stores and stores data in a set of append-only files.
-
-When append-only files:
-
-* Reach their maximum size, {brandname} creates a new file and starts writing to it.
-* Reach the compaction threshold of less than 50% usage, {brandname} overwrites the entries to a new file and then deletes the old file.
-
-.B+ trees
-
-To improve performance, append-only files in a `SoftIndexFileStore` are indexed using a **B+ Tree** that can be stored both on disk and in memory.
-The in-memory index uses Java soft references to ensure it can be rebuilt if removed by Garbage Collection (GC) then requested again.
+The following types of file-based cache stores exist in {brandname}:
+* Shared, where cache entries persist after a cluster restart operation. Examples include SQL, JDBC, and so on. 
+* Non-shared, where cache entries persist after a cluster restart operation, but in certain situations data loss might occur where individual nodes are restarted and the non-shared store contains stale key entries.  
 
-Because `SoftIndexFileStore` uses Java soft references to keep indexes in memory, it helps prevent out-of-memory exceptions.
-GC removes indexes before they consume too much memory while still falling back to disk.
+File-based cache stores should always be unique to each {brandname} instance.
+If you want to use the same persistent across a cluster, configure shared storage, such as a JDBC string-based cache store.
 
-You can configure any number of B+ trees with the `segments` attribute on the `index` element declaratively or with the `indexSegments()` method programmatically.
-By default {brandname} creates up to 16 B+ trees, which means there can be up to 16 indexes.
-Having multiple indexes prevents bottlenecks from concurrent writes to an index and reduces the number of entries that {brandname} needs to keep in memory.
-As it iterates over a soft-index file store, {brandname} reads all entries in an index at the same time.
+The following limitations exist with how a non-shared store interacts in a clustered environment: 
 
-Each entry in the B+ tree is a node.
-By default, the size of each node is limited to 4096 bytes.
-`SoftIndexFileStore` throws an exception if keys are longer after serialization occurs.
+* If you do not gracefully shutdown a cache, only the last node remains active and it must store all the cache's data. In-memory and eviction limitations might impact the amount of data a node can store. To prevent this issue, restart the last active node before you restart any of the other nodes. 
```

Comment:
This is all far too complex to expect a user to follow and does not hold true if deploying in a managed environment, such as k8s, as pods can be restarted by the scheduler.

https://github.com/infinispan/infinispan/pull/10360#discussion_r987828052

@dfitzmau I would hold off making any more changes to this PR for now, as I believe @wfink, @wburns and myself need to discuss this in person to determine what we should do going forward.

Reply:
Thanks, @ryanemerson . Yes, it's grown some legs. 

@wfink and @wburns . Please chime in on what can be simplified or removed.

Reply:
Hi @wfink . I sent a preview link in reply to your and Sangeeta's email. Did you receive it?